homeassistant/components/airzone_cloud/entity.py

```diff
@@ -132,6 +133,48 @@ async def _async_update_params(self, params: dict[str, Any]) -> None:
         self.coordinator.async_set_updated_data(self.coordinator.airzone.data())
 
 
+class AirzoneInstallationEntity(AirzoneEntity):
```

Comment:
Optional suggestion: Outside of this PR, you may want to consider using composition rather than inheritance given the number of double subclasses going on in this integration.